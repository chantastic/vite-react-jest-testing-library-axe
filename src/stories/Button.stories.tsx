import { ComponentMeta } from "@storybook/react";

import { Button } from "./Button";

export default {
  title: "Example/Button",
  component: Button,
  argTypes: {
    backgroundColor: { control: "color" },
  },
} as ComponentMeta<typeof Button>;

export const Primary = {
  args: {
    primary: true,
    label: "Button",
  },
};

export const Secondary = {
  args: {
    label: "Button",
  },
};

export const Large = {
  args: {
    size: "large",
    label: "Button",
  },
};

export const Small = {
  args: {
    size: "small",
    label: "Button",
  },
};

export const BackgroundColor = {
  args: {
    backgroundColor: "#C0FFEE",
    label: "Button",
  },
};
