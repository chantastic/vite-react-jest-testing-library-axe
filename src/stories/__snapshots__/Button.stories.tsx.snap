// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example/Button BackgroundColor smoke-test 1`] = `
<button type="button"
        class="storybook-button storybook-button--medium storybook-button--secondary"
        style="background-color: rgb(192, 255, 238);"
>
  Button
</button>
`;

exports[`Example/Button Large smoke-test 1`] = `
<button type="button"
        class="storybook-button storybook-button--large storybook-button--secondary"
>
  Button
</button>
`;

exports[`Example/Button Primary smoke-test 1`] = `
<button type="button"
        class="storybook-button storybook-button--medium storybook-button--primary"
>
  Button
</button>
`;

exports[`Example/Button Secondary smoke-test 1`] = `
<button type="button"
        class="storybook-button storybook-button--medium storybook-button--secondary"
>
  Button
</button>
`;

exports[`Example/Button Small smoke-test 1`] = `
<button type="button"
        class="storybook-button storybook-button--small storybook-button--secondary"
>
  Button
</button>
`;
